chainbase/src/main/java/org/tron/core/service/MortgageService.java

```diff
@@ -257,4 +260,18 @@ private void sortWitness(List<ByteString> list) {
     list.sort(Comparator.comparingLong((ByteString b) -> getWitnessByAddress(b).getVoteCount())
         .reversed().thenComparing(Comparator.comparingInt(ByteString::hashCode).reversed()));
   }
+
+  private long getOldReward(long beginCycle, long oldEndCycle, AccountCapsule accountCapsule) {
+    long cacheData = rewardCalService.getReward(accountCapsule.createDbKey(), beginCycle);
+    long reward = 0;
```

Comment:
This method is not optimized. If there is data in the cache, it should be returned directly.

Reply:
Phase I only calculates without using, cache enabled for the next release.